pkg/testing/log_watcher.go

```diff
@@ -0,0 +1,98 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package testing
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+)
+
+var _ Logger = &LogWatcher{}
```

Comment:
What is this used for?

Reply:
just to make sure it implements Logger interface, even if you're not using it anywhere (in case it's removed from test) check is performed